curves/mnt6_298/src/curves/g2.rs

```diff
@@ -20,7 +20,6 @@ impl CurveConfig for Config {
 
     /// COFACTOR =
     /// 226502022472576270196498690498308461791828762732602586162207535351960270082712694977333372361549082214519252261735048131889018501404377856786623430385820659037970876666767495659520
-    #[rustfmt::skip]
     const COFACTOR: &'static [u64] = &[
```

Comment:
This removal seems to be applied inconsistently

```suggestion
    const COFACTOR: &[u64] = &[
```